### PR TITLE
fix(dashy): active WA top border is now red

### LINKED ISSUE #3010: Dashy selected dashboard "Element" top border doesn't display red

### PR DESCRIPTION
**Pull request by jbrundage**
Cursor pointer indicates clickable widget adapter tabs
Fixes #3010

Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

### CODE REVIEW

File: packages/marshaller/src/ddl2/dashboard.css [hunk 1]
```diff
@@ -2,6 +2,13 @@
...
 
-.p-TabBar-tab.p-mod-current.active {
+.p-TabBar-tab.p-mod-current.active,
+.p-TabBar.active .p-TabBar-content {
     border-color:red;
-}
\ No newline at end of file
+}
+.p-TabBar .p-TabBar-content{
+    cursor: pointer;
```
Comment (GordonSmith) [inline packages/marshaller/src/ddl2/dashboard.css]: Not sure it should be a pointer for the entire tab bar (rather than an individual tab?)

Comment (jbrundage) [reply packages/marshaller/src/ddl2/dashboard.css]: It's only "pointer" while hovering over a tab (not the space to the right of the tabs).

**Comment by GordonSmith** [comment]
@jbrundage this is not working consistently in my testing...

**Comment by jbrundage** [comment]
@GordonSmith could you elaborate? I couldn't find any edge cases when I tested...

**Comment by GordonSmith** [comment]
1. Open a DockPanel with 3 widgets
2. Drag them around and stack them
3. After a while the horizontal colour gets out of sync

**Comment by GordonSmith** [comment]
Closing in favor of:  https://github.com/hpcc-systems/Visualization/pull/3029

**Pull request closed without merging**